config/clusters/2i2c-aws-us/neurohackademy.values.yaml

```diff
@@ -54,29 +54,21 @@ jupyterhub:
           node_selector:
             node.kubernetes.io/instance-type: r5.xlarge
             2i2c.org/community: neurohackademy
-        profile_options: &profile_options
-          interface:
-            display_name: Interface
-            choices:
-              tree:
-                display_name: Classic Notebook
-                slug: "tree"
-                kubespawner_override:
-                  default_url: "/tree"
-              lab:
-                display_name: JupyterLab
-                slug: "lab"
-                kubespawner_override:
-                  default_url: "/lab"
-              pull:
-                display_name: Pull curriculum repo and redirect to /lab
-                slug: "pull"
-                kubespawner_override:
-                  default_url: "/git-pull?repo=https%3A%2F%2Fgithub.com%2FNeuroHackademy-2023%2Fcurriculum&urlpath=lab&branch=main"
+          init_containers: &init_containers
+            # this container uses nbgitpuller to mount https://github.com/NeuroHackademy2024/curriculum for user pods
+            # image source: https://github.com/NASA-IMPACT/jupyterhub-gitpuller-init
+            - name: jupyterhub-gitpuller-init
+              image: public.ecr.aws/nasa-veda/jupyterhub-gitpuller-init:latest
+              env:
+                - name: TARGET_PATH
+                  value: "curriculum"
+                - name: SOURCE_REPO
+                  value: "https://github.com/NeuroHackademy2024/curriculum"
+                - name: SOURCE_BRANCH
+                  value: "main"
```

Comment:
@sgibson91, I believe you're missing mounting the home volume in the initiContainer:
- docs about it https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-initialization/#create-a-pod-that-has-an-init-container
- an existing example https://github.com/2i2c-org/infrastructure/blob/e179e314e28f9f08ba28c1b62ad1a93f98f7dcf5/config/clusters/pchub/common.values.yaml#L77-L90

Reply:
Thanks for looking. I did try that earlier, but added it back in and I still can't see the target folder in my workspace

```
(notebook) jovyan@jupyter-sgibson91:~$ ls -al
total 12
drwxr-xr-x 4 root   root   6144 Jul 23 09:28 .
drwxr-xr-x 1 root   root     21 Jul 23 12:12 ..
drwxr-xr-x 2 jovyan jovyan 6144 Jul 18 12:40 shared
drwxr-xr-x 2 jovyan jovyan 6144 Jul 18 12:40 shared-readwrite
```